content/applications/finance/accounting/taxes/avatax/avalara_portal.rst

```diff
@@ -0,0 +1,182 @@
+
+.. _avatax_portal:
+
+=======================
+Avalara (Avatax) portal
+=======================
+
+Avalara's (*AvaTax*) management console offers account management options including viewing/editing
+the transactions sent from Odoo to *AvaTax*, details on how the taxes are calculated, tax reporting,
+tax exemption management, and tax return resources.
+
+To access the console, first, navigate to either Avalara's `sandbox
+<https://sandbox.admin.avalara.com/>`_ or `production <https://admin.avalara.com/>`_ environment.
+This will depend on which type of account was set in the :doc:`integration <../avatax>`.
+
+.. _avalara_portal/transactions:
+
+Transactions
+============
+
+Next, to access transactions, click in the :guilabel:`Transactions` link on the main dashboard upon
+logging into the :ref:`console <avatax_portal>`. To manually access the *Transactions* page, while
```

Comment:
```suggestion
logging into the Avatax :ref:`console <avatax_portal>`.

To manually access the *Transactions* page, while
```

I suggest adding which webpage you are in- I wasn't sure if it was odoo or the other here.